_1327/minutes/templates/minutes_attachments.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'minutes_base.html' %}
+
+{% block title %}
```

Comment:
I guess we don't need that block as it is alredy declared like that in the base template `minutes_base.html`